src/base/bittorrent/sessionstatus.h

```diff
@@ -29,50 +29,43 @@
 #ifndef BITTORRENT_SESSIONSTATUS_H
 #define BITTORRENT_SESSIONSTATUS_H
 
-#include <libtorrent/session_status.hpp>
 #include <QtGlobal>
 
 namespace BitTorrent
 {
-    class SessionStatus
+    struct SessionStatus
     {
-    public:
-        SessionStatus(const libtorrent::session_status &nativeStatus);
```

Comment:
You are right, but I see a difference between an `#ifdef` inside of a function and when all the function is inside of `#fdef`: in the first case it is like a long `if` or `switch` and you do not know  what is after it, while in the second case you see that the whole function is valid only under certain condition.